### PR TITLE
Turn db_handler to a class (and other small things)

### PR DESCRIPTION
**Pull request by orelgueta**
The main difference is the db_handler is now a class. Had to make some changes to the telescope model as well, but they were quite small.
The problem, as usual, is that I can't test those changes. The db_handler tests pass (both Yaml and MongoDB), but I can't run the telescope model part. I am trying to do it on the wgs, but that turned out to be a nightmare (ran out of quota in the middle of creating the conda environment and now my conda is completely screwed up). 
@RaulRPrado, can you please run the other tests before you approve the merge?

### CODE REVIEW

**Comment by orelgueta** [comment]
Fixed my conda installation on the server and will run the tests myself (already found issues). Will fix and update this PR.

**Comment by RaulRPrado** [comment]
Looks good. I'll review it carefully soon, but probably not today.

**Comment by RaulRPrado** [comment]
Ok, everything is fine. You may merge it.

File: simtools/db_handler.py [hunk 1]
```diff
@@ -20,653 +22,879 @@
...
+    'writeModelFile',
+    'getModelFile',
+    'getFileMongoDB',
+    'writeFileFromMongoToDisk',
+    'copyTelescope',
+    'deleteQuery',
+    'updateParameter',
+    'addParameter',
+    'addNewParameter'
+]
```
Comment (RaulRPrado) [inline simtools/db_handler.py:38]: I think we should keep only DatabaseHandler in this list. All these functions are class methods, so they should not be here.

Comment (orelgueta) [reply simtools/db_handler.py:38]: Yea, that's a remnant from before I turned it into a class. Will fix now and merge.

**Pull request merged**